### ISSUE TITLE
2025-04-04, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#130]
# Today's Arxiv Papers

(1) [Sparse Autoencoders Learn Monosemantic Features in Vision-Language Models](http://arxiv.org/abs/2504.02821v1), 2025-04-03

(2) [Generative Evaluation of Complex Reasoning in Large Language Models](http://arxiv.org/abs/2504.02810v1), 2025-04-03

(3) [MegaMath: Pushing the Limits of Open Math Corpora](http://arxiv.org/abs/2504.02807v1), 2025-04-03

(4) [A Survey of Large Language Models in Mental Health Disorder Detection on Social Media](http://arxiv.org/abs/2504.02800v1), 2025-04-03

(5) [A Framework for Robust Cognitive Evaluation of LLMs](http://arxiv.org/abs/2504.02789v1), 2025-04-03

(6) [From Consumption to Collaboration: Measuring Interaction Patterns to Augment Human Cognition in Open-Ended Tasks](http://arxiv.org/abs/2504.02780v1), 2025-04-03

(7) [BT-ACTION: A Test-Driven Approach for Modular Understanding of User Instruction Leveraging Behaviour Trees and LLMs](http://arxiv.org/abs/2504.02779v1), 2025-04-03

(8) [MultiBLiMP 1.0: A Massively Multilingual Benchmark of Linguistic Minimal Pairs](http://arxiv.org/abs/2504.02768v1), 2025-04-03

(9) [How Deep Do Large Language Models Internalize Scientific Literature and Citation Practices?](http://arxiv.org/abs/2504.02767v1), 2025-04-03

(10) [Enhancing LLM Robustness to Perturbed Instructions: An Empirical Study](http://arxiv.org/abs/2504.02733v1), 2025-04-03

(11) [Why do LLMs attend to the first token?](http://arxiv.org/abs/2504.02732v1), 2025-04-03

(12) [ERPO: Advancing Safety Alignment via Ex-Ante Reasoning Preference Optimization](http://arxiv.org/abs/2504.02725v1), 2025-04-03

(13) [TeleMoM: Consensus-Driven Telecom Intelligence via Mixture of Models](http://arxiv.org/abs/2504.02712v1), 2025-04-03

(14) [The Hidden Space of Safety: Understanding Preference-Tuned LLMs in Multilingual context](http://arxiv.org/abs/2504.02708v1), 2025-04-03

(15) [LLM for Complex Reasoning Task: An Exploratory Study in Fermi Problems](http://arxiv.org/abs/2504.02671v1), 2025-04-03

(16) [Affordable AI Assistants with Knowledge Graph of Thoughts](http://arxiv.org/abs/2504.02670v1), 2025-04-03

(17) [Prompt Optimization with Logged Bandit Data](http://arxiv.org/abs/2504.02646v1), 2025-04-03

(18) [EmbodiedSense: Understanding Embodied Activities with Earphones](http://arxiv.org/abs/2504.02624v1), 2025-04-03

(19) [Multi-Mission Tool Bench: Assessing the Robustness of LLM based Agents through Related and Dynamic Missions](http://arxiv.org/abs/2504.02623v1), 2025-04-03

(20) [Exploring undercurrents of learning tensions in an LLM-enhanced landscape: A student-centered qualitative perspective on LLM vs Search](http://arxiv.org/abs/2504.02622v1), 2025-04-03

(21) [Multi-SWE-bench: A Multilingual Benchmark for Issue Resolving](http://arxiv.org/abs/2504.02605v1), 2025-04-03

(22) [LexPam: Legal Procedure Awareness-Guided Mathematical Reasoning](http://arxiv.org/abs/2504.02590v1), 2025-04-03

(23) [Rethinking RL Scaling for Vision Language Models: A Transparent, From-Scratch Framework and Comprehensive Evaluation Scheme](http://arxiv.org/abs/2504.02587v1), 2025-04-03

(24) [Language Models reach higher Agreement than Humans in Historical Interpretation](http://arxiv.org/abs/2504.02572v1), 2025-04-03

(25) [Leveraging LLM For Synchronizing Information Across Multilingual Tables](http://arxiv.org/abs/2504.02559v1), 2025-04-03

(26) [Exploring Individual Factors in the Adoption of LLMs for Specific Software Engineering Tasks](http://arxiv.org/abs/2504.02553v1), 2025-04-03

(27) [GPG: A Simple and Strong Reinforcement Learning Baseline for Model Reasoning](http://arxiv.org/abs/2504.02546v1), 2025-04-03

(28) [UNDO: Understanding Distillation as Optimization](http://arxiv.org/abs/2504.02521v1), 2025-04-03

(29) [A Memory-Augmented LLM-Driven Method for Autonomous Merging of 3D Printing Work Orders](http://arxiv.org/abs/2504.02509v1), 2025-04-03

(30) [ZClip: Adaptive Spike Mitigation for LLM Pre-Training](http://arxiv.org/abs/2504.02507v1), 2025-04-03

(31) [Inference-Time Scaling for Generalist Reward Modeling](http://arxiv.org/abs/2504.02495v1), 2025-04-03

(32) [MG-MotionLLM: A Unified Framework for Motion Comprehension and Generation across Multiple Granularities](http://arxiv.org/abs/2504.02478v1), 2025-04-03

(33) [Multimodal Fusion and Vision-Language Models: A Survey for Robot Vision](http://arxiv.org/abs/2504.02477v1), 2025-04-03

(34) [Retrieval-Augmented Purifier for Robust LLM-Empowered Recommendation](http://arxiv.org/abs/2504.02458v1), 2025-04-03

(35) [Cognitive Memory in Large Language Models](http://arxiv.org/abs/2504.02441v1), 2025-04-03

(36) [A Multi-Level Sentiment Analysis Framework for Financial Texts](http://arxiv.org/abs/2504.02429v1), 2025-04-03

(37) [Narrative Studio: Visual narrative exploration using LLMs and Monte Carlo Tree Search](http://arxiv.org/abs/2504.02426v1), 2025-04-03

(38) [Adapting Large Language Models for Multi-Domain Retrieval-Augmented-Generation](http://arxiv.org/abs/2504.02411v1), 2025-04-03

(39) [AnesBench: Multi-Dimensional Evaluation of LLM Reasoning in Anesthesiology](http://arxiv.org/abs/2504.02404v1), 2025-04-03

(40) [DaKultur: Evaluating the Cultural Awareness of Language Models for Danish with Native Speakers](http://arxiv.org/abs/2504.02403v1), 2025-04-03

(41) [The quasi-semantic competence of LLMs: a case study on the part-whole relation](http://arxiv.org/abs/2504.02395v1), 2025-04-03

(42) [CrystalFormer-RL: Reinforcement Fine-Tuning for Materials Design](http://arxiv.org/abs/2504.02367v1), 2025-04-03

(43) [ReuseDroid: A VLM-empowered Android UI Test Migrator Boosted by Active Feedback](http://arxiv.org/abs/2504.02357v1), 2025-04-03

(44) [Toward General and Robust LLM-enhanced Text-attributed Graph Learning](http://arxiv.org/abs/2504.02343v1), 2025-04-03

(45) [LearNAT: Learning NL2SQL with AST-guided Task Decomposition for Large Language Models](http://arxiv.org/abs/2504.02327v1), 2025-04-03

(46) [CoTAL: Human-in-the-Loop Prompt Engineering, Chain-of-Thought Reasoning, and Active Learning for Generalizable Formative Assessment Scoring](http://arxiv.org/abs/2504.02323v1), 2025-04-03

(47) [OmniCam: Unified Multimodal Video Generation via Camera Control](http://arxiv.org/abs/2504.02312v1), 2025-04-03

(48) [Improving Harmful Text Detection with Joint Retrieval and External Knowledge](http://arxiv.org/abs/2504.02310v1), 2025-04-03

(49) [Measurement of LLM's Philosophies of Human Nature](http://arxiv.org/abs/2504.02304v1), 2025-04-03

(50) [State-of-the-Art Translation of Text-to-Gloss using mBART : A case study of Bangla](http://arxiv.org/abs/2504.02293v1), 2025-04-03

(51) [Parallel Market Environments for FinRL Contests](http://arxiv.org/abs/2504.02281v1), 2025-04-03

(52) [LLM-Guided Evolution: An Autonomous Model Optimization for Object Detection](http://arxiv.org/abs/2504.02280v1), 2025-04-03

(53) [Reasoning Under 1 Billion: Memory-Augmented Reinforcement Learning for Large Language Models](http://arxiv.org/abs/2504.02273v1), 2025-04-03

(54) [Advancing Semantic Caching for LLMs with Domain-Specific Embeddings and Synthetic Data](http://arxiv.org/abs/2504.02268v1), 2025-04-03

(55) [MegaScale-Infer: Serving Mixture-of-Experts at Scale with Disaggregated Expert Parallelism](http://arxiv.org/abs/2504.02263v1), 2025-04-03

(56) [LLMs as Deceptive Agents: How Role-Based Prompting Induces Semantic Ambiguity in Puzzle Tasks](http://arxiv.org/abs/2504.02254v1), 2025-04-03

(57) [LLM Social Simulations Are a Promising Research Method](http://arxiv.org/abs/2504.02234v1), 2025-04-03

(58) [The Plot Thickens: Quantitative Part-by-Part Exploration of MLLM Visualization Literacy](http://arxiv.org/abs/2504.02217v1), 2025-04-03

(59) [LLM-Augmented Graph Neural Recommenders: Integrating User Reviews](http://arxiv.org/abs/2504.02195v1), 2025-04-03

(60) [More is Less: The Pitfalls of Multi-Model Synthetic Preference Data in DPO Safety Alignment](http://arxiv.org/abs/2504.02193v1), 2025-04-03

(61) [A Survey of Scaling in Large Language Model Reasoning](http://arxiv.org/abs/2504.02181v1), 2025-04-02

(62) [Subasa -- Adapting Language Models for Low-resourced Offensive Language Detection in Sinhala](http://arxiv.org/abs/2504.02178v1), 2025-04-02

(63) [Responsible Innovation: A Strategic Framework for Financial LLM Integration](http://arxiv.org/abs/2504.02165v1), 2025-04-02

(64) [OmniCellTOSG: The First Cell Text-Omic Signaling Graphs Dataset for Joint LLM and GNN Modeling](http://arxiv.org/abs/2504.02148v1), 2025-04-02

(65) [LL4G: Self-Supervised Dynamic Optimization for Graph-Based Personality Detection](http://arxiv.org/abs/2504.02146v1), 2025-04-02

(66) [Towards Interpretable Soft Prompts](http://arxiv.org/abs/2504.02144v1), 2025-04-02

(67) [On Simulation-Guided LLM-based Code Generation for Safe Autonomous Driving Software](http://arxiv.org/abs/2504.02141v1), 2025-04-02

(68) [Achieving Unanimous Consensus in Decision Making Using Multi-Agents](http://arxiv.org/abs/2504.02128v1), 2025-04-02

(69) [Efficient Model Selection for Time Series Forecasting via LLMs](http://arxiv.org/abs/2504.02119v1), 2025-04-02

(70) [LLMPi: Optimizing LLMs for High-Throughput on Raspberry Pi](http://arxiv.org/abs/2504.02118v1), 2025-04-02

(71) [PolyG: Effective and Efficient GraphRAG with Adaptive Graph Traversal](http://arxiv.org/abs/2504.02112v1), 2025-04-02

(72) [Exploring LLM Reasoning Through Controlled Prompt Variations](http://arxiv.org/abs/2504.02111v1), 2025-04-02

(73) [ScreenAudit: Detecting Screen Reader Accessibility Errors in Mobile Apps Using Large Language Models](http://dx.doi.org/10.1145/3706598.3713797), 2025-04-02

(74) [TiC-LM: A Web-Scale Benchmark for Time-Continual LLM Pretraining](http://arxiv.org/abs/2504.02107v1), 2025-04-02

(75) [ContrastScore: Towards Higher Quality, Less Biased, More Efficient Evaluation Metrics with Contrastive Evaluation](http://arxiv.org/abs/2504.02106v1), 2025-04-02

(76) [FlowDistill: Scalable Traffic Flow Prediction via Distillation from LLMs](http://arxiv.org/abs/2504.02094v1), 2025-04-02

(77) [Increasing happiness through conversations with artificial intelligence](http://arxiv.org/abs/2504.02091v1), 2025-04-02

(78) [Evolving Security in LLMs: A Study of Jailbreak Attacks and Defenses](http://arxiv.org/abs/2504.02080v1), 2025-04-02

(79) [Trapped by Expectations: Functional Fixedness in LLM-Enabled Chat Search](http://arxiv.org/abs/2504.02074v1), 2025-04-02

(80) [Aligned Better, Listen Better for Audio-Visual Large Language Models](http://arxiv.org/abs/2504.02061v1), 2025-04-02

(81) [Towards Operationalizing Heterogeneous Data Discovery](http://arxiv.org/abs/2504.02059v1), 2025-04-02

(82) [MageSQL: Enhancing In-context Learning for Text-to-SQL Applications with Large Language Models](http://arxiv.org/abs/2504.02055v1), 2025-04-02

(83) [From Prompts to Templates: A Systematic Prompt Template Analysis for Real-world LLMapps](http://arxiv.org/abs/2504.02052v1), 2025-04-02

(84) [Self-Resource Allocation in Multi-Agent LLM Systems](http://arxiv.org/abs/2504.02051v1), 2025-04-02

(85) [Towards Unified Referring Expression Segmentation Across Omni-Level Visual Target Granularities](http://arxiv.org/abs/2504.01954v1), 2025-04-02

(86) [The LLM Wears Prada: Analysing Gender Bias and Stereotypes through Online Shopping Data](http://arxiv.org/abs/2504.01951v1), 2025-04-02

(87) [OpenCodeReasoning: Advancing Data Distillation for Competitive Coding](http://arxiv.org/abs/2504.01943v1), 2025-04-02

(88) [Critical Thinking: Which Kinds of Complexity Govern Optimal Reasoning Length?](http://arxiv.org/abs/2504.01935v1), 2025-04-02

(89) [Review, Refine, Repeat: Understanding Iterative Decoding of AI Agents with Dynamic Evaluation and Selection](http://arxiv.org/abs/2504.01931v1), 2025-04-02

(90) [A thorough benchmark of automatic text classification: From traditional approaches to large language models](http://arxiv.org/abs/2504.01930v1), 2025-04-02

(91) [Is the Reversal Curse a Binding Problem? Uncovering Limitations of Transformers from a Basic Generalization Failure](http://arxiv.org/abs/2504.01928v1), 2025-04-02

(92) [Gen-C: Populating Virtual Worlds with Generative Crowds](http://arxiv.org/abs/2504.01924v1), 2025-04-02

(93) [Bridging the Linguistic Divide: A Survey on Leveraging Large Language Models for Machine Translation](http://arxiv.org/abs/2504.01919v2), 2025-04-02

(94) [Advancing AI-Scientist Understanding: Making LLM Think Like a Physicist with Interpretable Reasoning](http://arxiv.org/abs/2504.01911v1), 2025-04-02

(95) [Build Code Needs Maintenance Too: A Study on Refactoring and Technical Debt in Build Systems](http://arxiv.org/abs/2504.01907v1), 2025-04-02

(96) [STAR-1: Safer Alignment of Reasoning LLMs with 1K Data](http://arxiv.org/abs/2504.01903v1), 2025-04-02

(97) [TransientTables: Evaluating LLMs' Reasoning on Temporally Evolving Semi-structured Tables](http://arxiv.org/abs/2504.01879v1), 2025-04-02

(98) [Interpreting Emergent Planning in Model-Free Reinforcement Learning](http://arxiv.org/abs/2504.01871v1), 2025-04-02

(99) [From Code Generation to Software Testing: AI Copilot with Context-Based RAG](http://dx.doi.org/10.1109/MS.2025.3549628), 2025-04-02

(100) [Cross-Lingual Consistency: A Novel Inference Framework for Advancing Reasoning in Large Language Models](http://arxiv.org/abs/2504.01857v1), 2025-04-02
